repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/core/tracing/rewritetransid/system-model.cfg.xml

```diff
@@ -13,4 +13,5 @@
                       default="true"/>
         </destinations>
     </repose-cluster>
+    <tracing-header rewrite-header="true"/>
```

Comment:
Inconsistent with the other elements in `system-model` which are all nouns.  Not that that's a big deal at all, only my reasoning for going with `tracing-header` as the name of the element.  I'm fine either way. :sunglasses: 

I totally see your reasoning, though.  The three scenarios we want are:
- No tracing header
- Tracing header with no rewrite (default)
- Tracing header with rewrite

Ideally, I would expect them configured like this respectively:
- (nothing)
- `<tracing-header />`
- `<tracing-header rewrite="true" />`

The only problem is that default behavior, tho.  Something they were getting for nothing previously would now require an extra element.  I wonder if anyone would not even realize the connection between the header and that configuration item.  Even worse, downstream Repose instances could be impacted.  I'm just guessing as to why we might want to go with our original plan in the story.

@adrianjgeorge can we get a verdict, plz? :hammer: (why is there no gavel emoti?)